packages/terra-time-input/src/TimePipe.js

```diff
@@ -0,0 +1,33 @@
+function TimePipe(timeFormat) {
+  return function processConformedValue(conformedValue) {
+    if (!timeFormat || timeFormat.length < 0) {
+      return false;
```

Comment:
This file is no longer needed because the time input no longer depends on text-mask.